lib/shopify_cli/theme/dev_server.rb

```diff
@@ -108,6 +109,22 @@ def stop
           @syncer.shutdown
           WebServer.shutdown
         end
+
+        private
+
+        def find_theme(root, identifier)
+          return theme_by_identifier(root, identifier) if identifier
+          DevelopmentTheme.find_or_create!(ctx, root: root)
```

Comment:
It is an option to provide a dropdown menu to the client instead of defaulting to finding existing or creating new DevelopmentTheme